buildSrc/src/main/kotlin/Config.kt

```diff
@@ -103,7 +103,7 @@ object Versions {
         const val buildToolsVersion = "33.0.0"
         const val buildTools = "7.3.1" // https://maven.google.com/web/index.html?q=gradle#com.android.tools.build:gradle
         const val ndkVersion = "23.2.8568313"
-        const val r8 = "4.0.48" // See https://developer.android.com/build/kotlin-support
+        const val r8 = "8.0.34" // See https://developer.android.com/build/kotlin-support
```

Comment:
This R8 version is different from its minimum supported version defined in the changelog, is it ok?

Reply:
This version is also different than the one mentioned in the changelog

Reply:
This is the first version available from [Googles maven repository](https://maven.google.com/web/index.html?q=r8#com.android.tools:r8) which is above the minimum supported one specified in https://developer.android.com/build/kotlin-support, so just picked that 🤷 . We are only using it for test purposes so there is not strict reason to follow a specific version. But I can rewind it to `8.0.27` if you feel safer doing so.

Reply:
`8.0.27` does not seem to be available.